pom.xml

```diff
@@ -1950,7 +1950,11 @@
                 <param>com.puppycrawl.tools.checkstyle.ant.*</param>
                 <param>com.puppycrawl.tools.checkstyle.doclets.*</param>
               </targetTests>
-              <mutationThreshold>78</mutationThreshold>
+              <!-- till https://github.com/hcoles/pitest/issues/353 -->
```

Comment:
please move this comment down to be placed above certain `param` tag , not whole excludes.